Exoft.Gamification.Api.Data/Repositories/UserRepository.cs

```diff
@@ -18,26 +17,35 @@ public UserRepository(UsersDbContext context) : base(context)
 
         public async Task<User> GetUserAsync(Guid Id)
         {
-            var user = await DbSet
-                .Include(s => s.Roles)
-                    .ThenInclude(s => s.Role)
-                .Include(s => s.Achievements)
-                    .ThenInclude(s => s.Achievement)
-                .FirstOrDefaultAsync(i => i.Id == Id);
+            var user = await IncludeAll().FirstOrDefaultAsync(i => i.Id == Id);
```

Comment:
Havent noticed that.... it should be SingleOrDefaultAsync... Not only here but in other places to